notebooks/COS/Setup/Setup.ipynb

```diff
@@ -297,7 +297,7 @@
    "metadata": {},
```

Comment:
<p>I recommend implementing os.environ(), which allows you to set and modify environment variables within the script rather than relying on external configuration files like bash_profile.</p><p>This approach can help minimize errors or issues from manually interacting with the command line. Additionally, incorporating these steps directly within the notebook cell can make the process more streamlined and efficient.</p><p> </p><pre class="ql-syntax" spellcheck="false">import os

# Set the CRDS_PATH environment_variable
home_dir = os.environ["HOME"]
crds_path = os.path.join(home_dir, "crds_cache")&nbsp;&nbsp;
os.environ["CRDS_PATH"]=crds_path

# Set the CRDS_SERVER_URL environment_variable
crds_server_url = "https://hst-crds.stsci.edu"
os.environ["CRDS_SERVER_URL"] = crds_server_url

# Set the lref environment variable
lref = os.path.join(crds_path,"references/hst/cos")
os.environ["lref"] = lref

# Download reference files to the cache you set up.
!crds sync --contexts hst_cos_0347.imap --fetch-references

# To see the COS reference files downloaded
!ls $lref&nbsp;&nbsp;
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>